logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/DeleteAssetUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.logic.feature.asset
+
+import com.wire.kalium.logic.data.asset.AssetRepository
+import com.wire.kalium.logic.data.user.UserAssetId
+
+class DeleteAssetUseCase(private val assetRepository: AssetRepository) {
+
+    suspend operator fun invoke(assetKey: UserAssetId) {
```

Comment:
```suggestion
    suspend operator fun invoke(assetKey: AssetId) {
```

Does it makes sense to use AssetId instead of the specific avatar type ?